src/applications/disability-benefits/526EZ/components/create526EmailForm.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { connect } from 'react-redux';
+
+import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
+
+import { apiRequest } from '../../../../platform/utilities/api';
+import { requestStates } from '../../../../platform/utilities/constants';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
+
+class EmailForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      submissionStatus: requestStates.notCalled,
+      email: ''
+    };
+  }
+
+  updateEmail = (event) => {
+    this.setState({ email: event.target.value });
+  }
+
+  submitEmail = (event) => {
+    event.preventDefault();
+
+    this.setState({ submissionStatus: requestStates.pending });
+
+    /* apiRequest(`/form526_opt_in?email=${this.state.email}`, */
+    apiRequest('/form526_opt_in',
+      {
+        method: 'POST',
+        headers: {
+          'content-type': 'application/json'
+        },
+        body: JSON.stringify({ email: this.state.email })
+      },
+      () => {
+        this.setState({ submissionStatus: requestStates.succeeded });
+      },
+      () => {
+        this.setState({ submissionStatus: requestStates.failed });
+      });
+  }
+
+  render() {
+    const { loading, isLoggedIn, hasEVSSService } = this.props;
+    const { submissionStatus, email } = this.state;
+    if (loading) {
+      return null;
+    }
+
+    if (!isLoggedIn) {
+      return (
+        <AlertBox
+          isVisible
+          headline="To use our new tool, you’ll need to sign in with your premium My HealtheVet or DS Logon account"
+          content="If you don’t have a verified account, you can create an ID.me account to complete the verification process."
+          status="error"/>
+      );
+    }
+
+    if (!hasEVSSService) {
```

Comment:
Between this and `isLoggedIn`, we may want to actually use `RequiredLoginView` and then just check for an `isDataAvailable` prop here. But...that's also more complicated, and I don't really see a problem with doing things this way, especially in the short-term.

Reply:
I thought about it but decided against it for a few reasons:

1) This is super simple
2) `RequiredLoginView` doesn't gate based on the service list provided, so we'd still have to gate it manually
3) I don't know whether the `RequiredLoginView` component is already in the static page bundle, but if it isn't, I didn't want to add it